skgstat/models.py

```diff
@@ -98,7 +98,7 @@ def exponential(h, r, c0, b=0.0):
     r : float
         The effective range. Note this is not the range parameter! For the
         exponential variogram function the range parameter a is defined to be
-        :math:`a=\frac{r}{3}`. The effective range is the lag where 95% of the
+        :math:`a=\frac{r}{2}`. The effective range is the lag where 95% of the
```

Comment:
Long ago, that I wrote this stuff, but I think this was initially correct. the connection between effective range and range parameter should be 3a = r.